pom.xml

```diff
@@ -102,7 +102,7 @@
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
             <!-- see the Jenkins Enterprise compatibility note in the README before changing this version -->
-            <version>2.4</version>
+            <version>2.6.1.1</version>
```

Comment:
Does this have any implications for Enterprise compatibility?

Reply:
Looks like [this release](https://github.com/jenkinsci/mabl-integration-plugin#v0034-09042020) specifically addressed problems with the credentials plugin and Jenkins Enterprise. 

Were you able to follow the directions in the read me to verify that this new version works with Jenkins Enterprise?

Reply:
I validated being able to generate deployments in mabl from the plugin on the Jenkins Enterprise image.